books/free-programming-books-ko.md

```diff
@@ -237,8 +239,15 @@
 
 ### Rust
 
+* [러스트 코딩인사이트](https://coding-insight.com/docs/rust/start/rust)
```

Comment:
I think https://coding-insight.com/docs/category/rust/ is the top-level page for this